src/test/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksUtilityServiceImplTest.java

```diff
@@ -72,6 +75,44 @@ public void convertFieldArrayToMapFromValues() {
 		assertEquals("the value was not what we expected", PAYMENT_WORKS_FIELD_FROM_VALUE, results.get(PAYMENT_WORKS_FIELD_NAME));
 	}
 	
+	@Test
+	public void shouldVendorBeSentToPaymentWorks1() {
+		VendorDetail vendorDetail = buildVendorDetail();
+		vendorDetail.setActiveIndicator(true);
+		vendorDetail.getVendorHeader().setVendorTypeCode(CUVendorConstants.PROC_METHOD_DV);
+		assertTrue(paymentworksService.shouldVendorBeSentToPaymentWorks(vendorDetail));
+	}
+	
+	@Test
+	public void shouldVendorBeSentToPaymentWorks2() {
+		VendorDetail vendorDetail = buildVendorDetail();
+		vendorDetail.setActiveIndicator(true);
+		vendorDetail.getVendorHeader().setVendorTypeCode(CUVendorConstants.PROC_METHOD_PO);
+		assertTrue(paymentworksService.shouldVendorBeSentToPaymentWorks(vendorDetail));
+	}
+	
+	@Test
+	public void shouldVendorBeSentToPaymentWorks3() {
```

Comment:
This method could be renamed to something more meaningful like: activePcardVendorShouldNotBeSentToPaymentWorks